common/app/views/fragments/footerJavaScript.scala.html

```diff
@@ -74,8 +74,11 @@
                             $signInLinkElem.setAttribute('href', $signInLinkElem.getAttribute('href').replace('signin', 'public/edit/'));
                         }
                     }
-                    var $register = document.getElementsByClassName('js-profile-register')[0];
-                    $register.parentElement.removeChild($register);
+                    var $register = document.getElementsByClassName('js-profile-register');
```

Comment:
Minor note, but you could simply do `$register = document.getElementsByClassName('js-profile-register')[0]; if ($register) { …`. `[][0]` will result in `undefined`.